app/views/portal/clazzes/_materials.html.haml

```diff
@@ -1,20 +1,21 @@
 = javascript_include_tag "instructional_materials.js"
 = javascript_include_tag "share_material.js"
 - clazz_offerings = @portal_clazz.offerings.sort{|a,b| a.position<=>b.position}
+- material_text = t("material").pluralize.capitalize
 .maincontainer.pie
   .container
     .content
       = f.error_messages
     .right
       = render :partial => 'clazz_information', :locals => {:portal_clazz => portal_clazz}
-      .header Instructional Materials for #{portal_clazz.name}
+      .header Instructional #{material_text} for #{portal_clazz.name}
```

Comment:
The 'Instructional' should be removed from here. I'll probably just do this myself.